packages/components/src/components/FileUploader/FileUploader.stories.tsx

```diff
@@ -0,0 +1,156 @@
+import type { ComponentMeta, ComponentStory } from "@storybook/react";
+import React, { useState } from "react";
+import Uppy from "@uppy/core";
+import { FileUploader } from "./FileUploader";
+import { FileUploaderButton } from "./FileUploaderButton";
+
+export default {
+  component: FileUploader,
+  title: "Components/FileUploader",
+} as ComponentMeta<typeof FileUploader>;
+
+const Template: ComponentStory<typeof FileUploader> = (args) => (
+  <FileUploader {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  label: "Employment contract",
+  maxFileSize: "2MB",
+  children: (
+    <FileUploaderButton trailingIcon="ArrowUpTrayIcon" variant="secondary">
+      Upload
+    </FileUploaderButton>
+  ),
+};
+
+Default.parameters = {
+  chromatic: { viewports: [320, 1200] },
+};
+
+export const Loading = Template.bind({});
+Loading.args = {
+  label: "Visa",
+  fileName: "file_to_upload_with_a_long_name_v2.pdf",
+  progress: 50,
+  children: (
+    <FileUploaderButton trailingIcon="ArrowUpTrayIcon" variant="secondary">
+      Upload
+    </FileUploaderButton>
+  ),
+};
+
+Loading.parameters = {
+  chromatic: { viewports: [320, 1200] },
+};
+
+export const Success = Template.bind({});
+Success.args = {
+  label: "Passport scan",
+  fileSize: "2MB",
+  fileName: "my_passport_scap.pdf",
+  fileUrl: "http://file-path/file.pdf",
+  children: (
+    <FileUploaderButton trailingIcon="ArrowUpTrayIcon" variant="secondary">
+      Upload
+    </FileUploaderButton>
+  ),
+};
+
+Success.parameters = {
+  chromatic: { viewports: [320, 1200] },
+};
+
+export const SuccessWithoutFileInfo = Template.bind({});
+SuccessWithoutFileInfo.args = {
+  label: "Passport scan",
+  fileUrl: "http://file-path/file.pdf",
+  children: (
+    <FileUploaderButton trailingIcon="ArrowUpTrayIcon" variant="secondary">
+      Upload
+    </FileUploaderButton>
+  ),
+};
+
+SuccessWithoutFileInfo.parameters = {
+  chromatic: { viewports: [320, 1200] },
+};
+
+export const WithError = Template.bind({});
+WithError.args = {
+  label: "Registration confirmation",
+  fileName: "file_with_error_too_large.pdf",
+  fileSize: "7MB",
+  errorMessage: "File size is too large. You can upload files up to 5MB.",
+  children: (
+    <FileUploaderButton trailingIcon="ArrowUpTrayIcon" variant="secondary">
+      Upload
+    </FileUploaderButton>
+  ),
+};
+
+WithError.parameters = {
+  chromatic: { viewports: [320, 1200] },
+};
+
+export const WithUppy = (): React.ReactElement => {
+  const uppy = new Uppy();
```

Comment:
As much as I appreciate this effort a ton I wouldn't add uppy as a dependency on Pluto. Is that intended to stay?

Reply:
sure! It'll still be installed when storybook is deployed, right? Anyway lgtm